src/Management/src/Endpoint/SpringBootAdminClient/SpringBootAdminClientHostedService.cs

```diff
@@ -49,6 +49,11 @@ public async Task StartAsync(CancellationToken cancellationToken)
     {
         SpringBootAdminClientOptions clientOptions = _clientOptionsMonitor.CurrentValue;
 
+        if (clientOptions.Url == null)
+        {
+            throw new InvalidOperationException("Spring Boot Admin Server Url must be provided in options.");
+        }
```

Comment:
Is it safe to throw here? Running from a hosted service (background thread), would that crash the app or swallow the error? What's the experience when trying this from a sample?

Reply:
This change results in an app crash, which feels right (at least for a local dev) since we don't ever retry this operation and you need to restart to register.

Without the change the app runs but obviously never registers, and you see this in the logs:
```
fail: Steeltoe.Management.Endpoint.SpringBootAdminClient.SpringBootAdminClientHostedService[0]
      Error connecting to Spring Boot Admin Server: An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set.
      System.InvalidOperationException: An invalid request URI was provided. Either the request URI must be an absolute URI or BaseAddress must be set.
```

Reply:
The duplicate message is due to this line in `SpringBootAdminClientHostedService`:
```c#
_logger.LogError(exception, "Error connecting to Spring Boot Admin Server: {Message}", exception.Message);
```
It should just be:
```c#
_logger.LogError(exception, "Error connecting to Spring Boot Admin Server.");
```
Can you fix the logging in this file?

Reply:
Yeah, I'll fix the log message. What do you think about keeping the new exception?

Reply:
I'd have to try it out to see what happens. But you probably have a sample ready, so can we discuss it during standup?